README.md

```diff
@@ -1,45 +1,73 @@
-# cloud-platform-terraform-template
+# cloud-platform-terraform-opensearch-alert
 
-[![Releases](https://img.shields.io/github/v/release/ministryofjustice/cloud-platform-terraform-template.svg)](https://github.com/ministryofjustice/cloud-platform-terraform-template/releases)
+[![Releases](https://img.shields.io/github/v/release/ministryofjustice/cloud-platform-terraform-opensearch-alert.svg)](https://github.com/ministryofjustice/cloud-platform-terraform-opensearch-alert/releases)
+
+This Terraform module creates an OpenSearch alert for detecting specific conditions and sends notifications to a Slack channel. It allows users to configure various aspects of the alert, including the query, triggers, and Slack channel settings.
```

Comment:
Can you specify explicitly that this is only to create alerts for user application logs. It's also be worth creating a ticket, to add a feature in future to allow alerts to be created with this module for our modsec opensearch cluster

Reply:
updated the readme. I think we can add the variable to the data block. 

```
data "aws_opensearch_domain" "live_app_logs" {
  domain_name = var.domain_name
}

data "aws_iam_role" "os_access_role_app_logs" {
  name = var.aws_iam_role
}
```

We can keep the default variable value to our user with `var.domain_name = "cp-live-app-logs" `and `var.aws_iam_role = "opensearch-access-role-app-logs"`

If we would like to use it for modsec opensearch cluster, we can change the variable respectively in our terraform code